### PR TITLE
Report extension status for disallowed downgrade

### PR DESCRIPTION
**Pull request by uiri**
Fix a bug in #1057

### CODE REVIEW

File: azurelinuxagent/ga/exthandlers.py [hunk 1]
```diff
@@ -392,6 +392,8 @@ def handle_enable(self, ext_handler_i):
            old_ext_handler_i.version_gt(ext_handler_i):
             msg = "Downgrade is not allowed. Skipping install and enable."
```
Comment (boumenot) [inline azurelinuxagent/ga/exthandlers.py]: Is this message accurate based on your change?

File: azurelinuxagent/ga/exthandlers.py [hunk 2]
```diff
@@ -392,6 +392,8 @@ def handle_enable(self, ext_handler_i):
...
             ext_handler_i.logger.error(msg)
```
Comment (boumenot) [inline azurelinuxagent/ga/exthandlers.py]: This should be info or a verbose, but not an error.

File: azurelinuxagent/ga/exthandlers.py (hunk 1 above)
Comment (uiri) [reply azurelinuxagent/ga/exthandlers.py]: Yes. This block hits a return statement at line 399. The install and enable logic is from lines 401 to 420 in this function.

**Comment by boumenot** [comment]
LGTM.  Any more concerns, or are you ready to merge?

**Pull request closed without merging**